lms/lib/comment_client/user.py

```diff
@@ -31,7 +31,9 @@ class User(models.Model):
     def from_django_user(cls, user):
         return cls(id=str(user.id),
                    external_id=str(user.id),
-                   username=user.username)
+                   username=user.username,
+                   first_name = user.first_name,
```

Comment:
Why aren't we using `display_name` from `api_integration` here?

Reply:
@nasirhjafri its a comment_client user, here we are not accessing api-integration.

Reply:
hmm, then why don't we create and pass `display_name` from here instead of passing first and last name?